tests/unit/utils/test_metadata_utils.py

```diff
@@ -63,3 +65,47 @@ def test_sort_prog_descriptions_returns_default_value(self) -> None:
             }
             not_in_config_value = {'source': 'test', 'text': 'I am a test'}
             self.assertEqual(_sort_prog_descriptions(mock_config, not_in_config_value), len(mock_config))
+
+
+class UneditableTableDescriptionTest(unittest.TestCase):
+    def setUp(self) -> None:
+        pass
+
+    def test_table_desc_match_rule_schema_only(self) -> None:
+        with local_app.app_context():
```

Comment:
Will these tests pass if you remove `with local_app.app_context():`? I believe we only have to include that when the code we are testing tries to access `app.config`.

Reply:
yeah you are right, we may not need this.